finetune_adapter.py

```diff
@@ -174,11 +174,12 @@ def get_batch(fabric: L.Fabric, data: list):
     input_ids = [data[i]["input_ids"].type(torch.int64) for i in ix]
     labels = [data[i]["labels"].type(torch.int64) for i in ix]
 
-    max_len = max(len(s) for s in input_ids)
+    if data.device.type != "xla":
+        max_seq_length = max(len(s) for s in input_ids)
```

Comment:
```suggestion
    max_len = max(len(s) for s in input_ids) if data.device.type != "xla" else max_seq_length
```
I propose this instead of modifying the max_seq_length that is also global